src/topology/algebra/polynomial.lean

```diff
@@ -203,6 +203,35 @@ begin
     rw [←hcd, noroots, card_zero], },
 end
 
+/-- The coefficients of the monic polynomials of bounded degree with bounded roots are
+uniformely bounded. -/
+lemma coeff_bdd_of_roots_le (B : ℝ) (d : ℕ) (f : F →+* K) :
+   ∃ C, ∀ p : F[X], p.monic → splits f p → p.nat_degree ≤ d → (∀ z ∈ (map f p).roots, ∥z∥ ≤ B) →
+   ∀ i, ∥(map f p).coeff i∥ ≤ C :=
```

Comment:
Why not prove a stronger result with `C = B^d * 2^(d+1)`? Then it's also clear that `C` doesn't depend on `f`, and you can move the arguments to the left of `:`.

Reply:
Well, at the start, I only needed to know that the bound existed for my main result. In fact, I proved the explicit bound in a previous version bu the proof was quite longer. I see your point though...

Reply:
All right, it turns out if you do it right, the proof gets a lot shorter. Thanks for the suggestion! 

Note that I had to add the hypothesis `0 ≤ B` to make it work. When I wrote `coeff_le_of_roots_le` (which is just above in the file), I did not used this condition. If `B < 0`, then the polynomial is constant and thus equal to 1 and the result is trivially true. Still it takes about 15 lines to deal with this case and I am not sure it was a good idea after all.

Reply:
Please try to avoid unnecessary assumptions. We can try to improve the proof if it is too slow.

Reply:
Ok. So I'll do like in the proof of `coeff_le_of_roots_le` and remove the assumption that `0 ≤ B`. There is probably a way to more efficient.